boards/stm32f3discovery/src/main.rs

```diff
@@ -251,15 +251,15 @@ unsafe fn set_pin_primary_functions(
         pin.set_mode(stm32f303xc::gpio::Mode::AnalogMode);
     });
 
-    // channel 3
-    gpio_ports.get_pin(PinId::PA02).map(|pin| {
-        pin.set_mode(stm32f303xc::gpio::Mode::AnalogMode);
-    });
+    // // channel 3
+    // gpio_ports.get_pin(PinId::PA02).map(|pin| {
+    //     pin.set_mode(stm32f303xc::gpio::Mode::AnalogMode);
+    // });
```

Comment:
This seems like an unrelated change

Reply:
Those are the pins used to test the second uart, reverted.